### ISSUE TITLE
Hero5 Black hangs more than it works whenever I python3 *py.

### ISSUE THREAD

**Issue by garydelarosausername** [KonradIT/GoProStream#17]
The script fails more than it works, might be unstable wireless connectivity. The connection coming from the laptop might be too much for the cam? I don't know. I 've tried 2.4m to 10m, none of them works more than it fails. 

Help please? I'm looking into it too.

Thanks!

**Comment by KonradIT** [comment]
Not a problem for me, make sure its on and with wifi in APP mode and with the latest FW

**Comment by garydelarosausername** [comment]
The camera is on and updated. It's also in Video mode. You mean Video mode and not APP mode right?

**Comment by garydelarosausername** [comment]
Nope. Script freezed my camera again. What I do is connect to camera's wifi, run the script - python3 *py, and then hope that it works. Camera's updated too.

**Comment by KonradIT** [comment]
modify the script flags.

**Comment by KonradIT** [comment]
Works on my H5B: https://i.imgur.com/QxMDxMm.png

My flags:

```VERBOSE=False
## Sends Record command to GoPro Camera, must be in Video mode!
RECORD=False
##
## Saves the feed to a custom location
SAVE=False
```

**Comment by garydelarosausername** [comment]
Oh no! I forgot! Sorry! I was talking about Record mode.

**Comment by garydelarosausername** [comment]
@KonradIT I wanna let you know that the Preview mode works most of the time! I ran it while I had my GoPro strapped on my head. It worked great! I'll post clips of it on my Instagram and YouTube! Thanks a lot! :ok_hand:

**Comment by thwyster** [comment]
Hi Konrad, i`m try use this script to connect my Hero5Black, but when try run the script, I received this message: "ffplay 'is not recognized as an internal command
or external, an operational program or a batch file."

do you have any idea??

**Comment by thwyster** [comment]
I was able to run the screen using the following command (windows 7): ffplay -fflags nobuffer -f: v mpegts -probesize 8192 udp: //@10.5.5.9: 8554 ...

Would you like to know if it is possible to capture the image of 2 Gopro?
and what steps would be required for that purpose, I tried running GoproStream.py with another ip and port, but when I run the second screen it gets stuck.

Any idea?
